### ISSUE TITLE
Dynasty-Chapters: chapters would disappear from library after finishing download

### ISSUE THREAD

**Issue by yuudechi** [tachiyomiorg/tachiyomi-extensions#11543]
### Source information

Dynasty 1.2.16

### Steps to reproduce

1. Use "Dynasty Chapters" source, add and download any individual chapter.
2. Let the download finish.
3. Once finished, if you try to read the chapter, it will try to fetch the live images from the site instead.

### Expected behavior

I should be able to read the downloaded chapters without issue.

### Actual behavior

Once the download is finished, the chapter will disappear from the library (you might have to go back to the main library, then click on the chapter again to have it fully disappear). You won't be able to interact with the downloaded chapter either. The only way to delete it would be to remove the chapter from the library, and clicking the "Delete" button from "Delete downloaded chapters?" pop up.

The downloaded chapter is on the phone though. I've noticed that the naming scheme is different from previous Tachiyomi versions however.

### Tachiyomi version

Stable 0.13.3 and Preview r4266

### Android version

Android 12

### Other details

This bug is on both the stable 0.13.3 and preview r4266 versions. Deleting app data and reinstalling the extension had no effects.

So far, other extensions I've used don't have this issue.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by joseph619** [comment]
This isn't extension issue, wait for next Stable update for fix.
